projects/envoy/find_corpus.py

```diff
@@ -1,15 +1,23 @@
-#!/usr/bin/python
-import os, sys, re
-fuzzer_target = sys.argv[1].split('/')
-directory, fuzz_test = fuzzer_target[:-1], fuzzer_target[-1]
-directory_string = "/".join(directory)
-path = "../envoy/" + directory_string + "/BUILD"
-with open(path, "r") as f:
-        searchlines = f.readlines()
-        for i, line in enumerate(searchlines):
-                if fuzz_test in line: 
-                    for l in searchlines[i:]:
-                        if "corpus =" in l:
-                              corpus_path = l 
-                              break
+#!/usr/bin/pxython
+
+import os
+import sys
+import re
+
+fuzzer_target = sys.argv[1].split("/")
```

Comment:
There's still inconsistency in this file in the use of `'` vs. '"`. Please stick to one consistent style.